docs/source/troubleshooting/troubleshooting-misc.adoc

```diff
@@ -93,3 +93,35 @@ Workaround: Delete the existing {project} VM and start again.
 $ minishift delete --force
 $ minishift start
 ----
+
+[[Remove-password-from-keychain]]
+== Removing subscription password from OS native keychain
+
+This only applies if you are using a subscription capable distro like RHEL with {project}. 
```

Comment:
s/This/These steps/ 
s/you are using a subscription capable distro like RHEL with {project}./you are using a subscription capable ISO image like Red Hat Enterprise Linux with {project} which needs registration using username/password./